datalogtool/src/main/generate/WPILibVersion.cpp.in

```diff
@@ -1,4 +1,4 @@
-/*
+/**
  * Autogenerated file! Do not manually edit this file. This version is regenerated
  * any time the publish task is run, or when this file is deleted.
  */
```

Comment:
Are these actually doc comments for `GetWPILibVersion()`?

Reply:
Yes.